src/server/pfs/cmds/cmds.go

```diff
@@ -433,6 +434,7 @@ Files can be read from finished commits with get-file.`,
 	}
 	addShardFlags(listFile)
 	listFile.Flags().BoolVar(&unsafe, "unsafe", false, "use this flag if you need to list files written in the current commit; this operation will race with concurrent writes")
+	listFile.Flags().StringVarP(&fromCommit, "from-commit", "f", "", "only list files that are written since a given commit")
```

Comment:
We use `"from"` not `"from-commit"` in `GetFile` could we use that here as well?